recipe/meta.yaml

```diff
@@ -27,7 +28,6 @@ requirements:
     - pip
     - poetry-core >=1.0.0a9
     - python
-    - setuptools
```

Comment:
I am curious to know why `setuptools` was removed

Reply:
@skupr-anaconda mentioned this in the 0.10.2 feedstock here:
https://github.com/AnacondaRecipes/tomlkit-feedstock/pull/8

And cited the pyproject.toml here:
https://github.com/sdispater/tomlkit/blob/e15bb79401517ca28013454662741ca9b0646fab/pyproject.toml#L61

Reply:
@varlackc Yeah, there is another build backend - `poetry`, not `setuptools`